sheet_to_triples/run.py

```diff
@@ -42,6 +42,11 @@ def use_non_uniques(self, old_transforms):
         for tf in old_transforms:
             self.non_unique.update(tf.get_non_uniques(self.ns))
 
+    def set_terms(self, triples):
+        if self.model:
+            self.model.terms[:] = ()
+            rdf.update_model_terms(self.model, triples)
```

Comment:
`rdf.update_model_terms`

```
def update_model_terms(model, triples):
    model['terms'].extend(
        dict(subj=str(s), pred=str(p), obj=str(o)) for s, p, o in triples)
```

I don't think you can `extend` a tuple?

Reply:
Wait, this is a list slicing operation isn't it, just replacing the contents with a 0 arg tuple. Never mind!

Reply:
Odd bit of python syntax, distinguishing 'clear referenced list' from 'replace variable', which likely doesn't matter here anyway.
```python
some_list = []  # make a new list, drop reference to anything some_list was before
# vs
some_list[:] = iter(()) # remove the contents of some_list, replacing with values from (empty) iter
```